about.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+ <link rel="stylesheet" href="style.css">
+ <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=lato">
+ <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+ <script src="https://kit.fontawesome.com/7967e81f77.js" crossorigin="anonymous"></script>
+  <title>Document</title>
+</head>
+<body>
+  <main>
+    <section class="about">
+  <header class="nav-header">
+    <nav class="navItems">
+      <a class="logo navbar-brand" href="/index.html"> <img src="icons/kamzzylearns.png" alt="logo"> </a>
+    <ul class="nav-menu">
+      <li><a class="about-nav nav-link" href="/index.html">Home</a></li>
+     <li><a class="about-nav nav-link" href="/about.html">About</a></li>
```

Comment:
@Ismailco kindly explain what functionality you expect from the nav links because currently the home links to the index page about links to the about page while the logo links to the index page as well

Reply:
The links are not working properly, you can change them to ```kamzzy-learns/index.html``` and ```kamzzy-learns/about.html```
![notWorking](https://user-images.githubusercontent.com/25227022/164535159-a9f32d65-5549-4309-aec7-eabb6f2a346e.gif)

Reply:
This issue was not addressed 😥 Please be sure to check your <a> tags and their hrefs because that is causing the issue. 
As you can see in the following image, this is the URL path that the browser looks for when the user clicks on the about `<a>` tag. The file is not found because the path is incorrect. 

![image](https://user-images.githubusercontent.com/15807343/164550618-1460eb36-900a-4abf-9841-e9919e154887.png)

- The proper path is `https://kamzzy.github.io/kamzzy-learns/about.html` 

![a](https://user-images.githubusercontent.com/15807343/164550948-bda63f3f-b81d-4644-92b2-6d1db3569c98.jpg)

- This means that the bug is on your about <a> tag 🤓 Kindly change it to `/kamzzy-learns/about.html` and will work 🙌

![image](https://user-images.githubusercontent.com/15807343/164551163-6d68cf96-8b75-46f4-9ad5-3c29e75ca84a.png)